python/pyarrow/_dataset_parquet.pyx

```diff
@@ -715,6 +714,9 @@ cdef class ParquetFragmentScanOptions(FragmentScanOptions):
     decryption_config : pyarrow.dataset.ParquetDecryptionConfig, default None
         If not None, use the provided ParquetDecryptionConfig to decrypt the
         Parquet file.
+    decryption_properties : pyarrow.dataset.FileDecryptionProperties, default None
```

Comment:
Is this correct? I only see C++ class FileDecryptionProperties under parquet namespace.

Reply:
Nice catch! Changed to:
```python
  decryption_properties : pyarrow.parquet.FileDecryptionProperties, default None
```

Reply:
I'll wait for the CI and merge.